### ISSUE TITLE
implement compute kernel for ImageDecoder

### ISSUE THREAD

**Issue by liqunfu** [microsoft/onnxruntime#17593]
### Describe the feature request

ImageDecoder has been add to ONNX 1.15.0: https://github.com/onnx/onnx/pull/5294. Need to implement compute kernel once ONNX 1.15.0 is in ORT (https://github.com/microsoft/onnxruntime/pull/17125).
This issue shall include re-enable test of ImageDecoder op at 
onnxruntime/test/testdata/onnx_backend_test_series_filters.jsonc

### Describe scenario use case

onnx_test_runner shal pass ONNX test cases for the ImageDecoder op

**Comment by jantonguirao** [comment]
I'd like to ask for an update on this request.
Is it committed to be added to 1.17 release?

**Comment by snnn** [comment]
No.  To implement the operator, we need to integrate codecs libraries to ONNX Runtime. We may either use a platform independent solution, like opencv, for all platforms, or develop separated code for each platform. Both have pros and cons. Haven't made a decision yet.

**Comment by ShirasawaSama** [comment]
> No. To implement the operator, we need to integrate codecs libraries to ONNX Runtime. We may either use a platform independent solution, like opencv, for all platforms, or develop separated code for each platform. Both have pros and cons. Haven't made a decision yet.

Sorry to bother you, but has there been any progress on this? If ORT has decided which image decoding library to use, I can help with the implementation.
